bitrise.yml

```
@@ -0,0 +1,87 @@
+---
+format_version: '8'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+trigger_map:
+  - push_branch: main
+    workflow: app-store-build
+  - pull_request_target_branch: develop
+    workflow: qa-build
+workflows:
+  app-store-build:
+    steps:
+      - xcode-archive@4:
+          inputs:
+            - project_path: '$BITRISE_PROJECT_PATH'
+            - scheme: '$BITRISE_SCHEME'
+            - distribution_method: app-store
+            - configuration: RELEASE
+            - xcconfig_content: CURRENT_PROJECT_VERSION = $BITRISE_BUILD_NUMBER
+            - export_method: app-store
+      - deploy-to-bitrise-io@2: {}
+      - deploy-to-itunesconnect-deliver@2:
+          inputs:
+            - app_id: '$APP_ID'
+            - connection: 'off'
+            - api_key_path: '$BITRISEIO_APP_STORE_CONNECT_API_KEY_URL'
+            - api_issuer: '$API_KEY_ISSUER_ID'
+      - github-release@0:
+          inputs:
+            - username: '$GITHUB_RELEASE_USERNAME'
+            - name: 1.0+$BITRISE_BUILD_NUMBER
```

```suggestion
            - name: 8.0+$BITRISE_BUILD_NUMBER
```

My mistake for not updating this on Bitrise's end, but in App Store Connect (and the project) we're referencing the major version as `8`.